### ISSUE TITLE
DO NOT EXECUTE

### ISSUE THREAD

**Issue by PetrTech** [mevev/game#1]
Your friend probably got token logged and is now sending out links, don't download this. This is malware...

**Comment by AZMindroma** [comment]
I can confirm this. This was sent to a friend just recently from a compromised account. Make sure to report the user to GitHub Support as soon as possible to prevent further compromised accounts.

**Comment by PetrTech** [comment]
> I can confirm this. This was sent to a friend of mine just recently from a compromised account. Make sure to report the user to GitHub Support as soon as possible to prevent further compromised accounts.

Yes! Reported already...

If I got the chance to extract this and look at the code, I DEFINITELY WOULD.

**Comment by Backtwice** [comment]
Understandable. Thank God I read this before running.

**Comment by Backtwice** [comment]
Thank you guys too for letting everyone know.

**Comment by PetrTech** [comment]
> Thank you guys too for letting everyone know.

Yeah don't click and download random things, always check the source and file size before actually running the file

**Comment by PetrTech** [comment]
Also I'm sure this issue will get deleted soon to cover the creator's tracks

**Comment by adorably** [comment]
> Yeah don't click and download random things, always check the source and file size before actually running the file

(⌣́_⌣̀)

**Comment by Backtwice** [comment]
> Also I'm sure this issue will get deleted soon to cover the creator's tracks

Yeah I reported his account for posting malware.

**Comment by PetrTech** [comment]
You should never trust stuff by NEW PEOPLE with NEW REPOSITORIES without Open Sourcing it.
Especially if it's sent by your friend on DISCORD. Discord is the place where most malware gets sent around.

If you got affected by this, change your password RIGHT NOW
Enabling 2FA would also work after changing your pass, tho it won't reset your token like a pass reset would. But you can do this for the advanced security.

**Comment by PetrTech** [comment]
I am going to wait for this guy to get banned, tho these ppl are creating accounts like every second and uploading the malware on those.

**Comment by Backtwice** [comment]
> You should never trust stuff by NEW PEOPLE with NEW REPOSITORIES without Open Sourcing it. Especially if it's sent by your friend on DISCORD. Discord is the place where most malware gets sent around.
> 
> If you got affected by this, change your password RIGHT NOW Enabling 2FA would also work after changing your pass, tho it won't reset your token like a pass reset would. But you can do this for the advanced security.

Yep, that's what I just said on a 30,000 member Discord. 30K less people who will fall for this.

**Comment by PetrTech** [comment]
![image](https://user-images.githubusercontent.com/55279432/151862932-6d6b1d9e-27c3-4cef-a757-daa18bbab2c0.png)

Perfect image to add to this thread lol

**Comment by Backtwice** [comment]
> ![image](https://user-images.githubusercontent.com/55279432/151862932-6d6b1d9e-27c3-4cef-a757-daa18bbab2c0.png)
> 
> Perfect image to add to this thread lol

LMFAO

**Comment by PetrTech** [comment]
I **don't** exactly know what this malware does, but it sure is malware.

**Comment by Backtwice** [comment]
Yes.

**Comment by PetrTech** [comment]
It's been 30 minutes already. Microsoft needs to ban this guy now before this spreads even more.

**Comment by Backtwice** [comment]
I think there are a lot of other accounts to ban right now.

**Comment by PetrTech** [comment]
Probably

**Comment by PetrTech** [comment]
I hope this actually helped everyone.

Also how did you find the thread? because if you click on the sent link the game will download right away.
Did you search for this user too before downloading?
That's what people should do before clicking free vbucks and robux links.

**Comment by Backtwice** [comment]
> I hope this actually helped everyone.
> 
> Also how did you find the thread? because if you click on the sent link the game will download right away. Did you search for this user too before downloading? That's what people should do before clicking free vbucks and robux links.

Yep.

**Comment by Backtwice** [comment]
![image](https://user-images.githubusercontent.com/95854905/151865279-c541c0d6-a1ed-4cb7-ad04-b7649ddb5634.png)

**Comment by Backtwice** [comment]
Oops wrong screenshot

**Comment by Backtwice** [comment]
![image](https://user-images.githubusercontent.com/95854905/151865353-e60df7cb-2e0b-472a-94c1-83be81086d5a.png)
